DAS/tests/fake_module_tester.py

```diff
@@ -2,8 +2,8 @@
 import argparse
 import json
 import paho.mqtt.client as mqtt
-from MockSensor import MockSensor
-import enum_topics
+from DAS.utils.MockSensor import MockSensor 
```

Comment:
Same with this, is there anyway we can just do `from DAS.utils import MockSensor`?